duckduckgo/iOS
# tweak major tracker dax dialog copy

Task/Issue URL: https://app.asana.com/0/392891325557410/1194920959117657
Tech Design URL:
CC:

**Description**:

Tweak the copy for major tracker networks.

**Steps to test this PR**:
1. Fresh install, onboarding, visit a major tracker (facebook.com or google.com)
1. New copy should be displayed.



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
